specification/columns/skupricedetails.md

```diff
@@ -0,0 +1,54 @@
+# SKU Price Details
+
+The **SKU Price Details** column represents a list of relevant properties shared by all resources having the same [**SKU Price ID**](#skupriceid). These properties enable a practitioner to more easily calculate metrics such as total units of a service when it is not directly billed in those units (e.g. cores) and thus enables FinOps capabilities such as unit economics. They also help a practitioner understand the specifics of a **SKU Price ID**.
+
+
+The _SkuPriceDetails_ column adheres to the following requirements:
+
+* The _SkuPriceDetails_ column MUST be in [_KeyValueFormat_](#key-valueformat).
+* The key for a property SHOULD be formatted in [PascalCase](#glossary:pascalcase).
+* The properties (both keys and values) contained in the _SkuPriceDetails_ column MUST be shared across all charges having the same _SkuPriceId_.
+* The key for a property SHOULD remain consistent across comparable SKUs having that property and the values for this key SHOULD remain in a consistent format.
+* The _SkuPriceDetails_ column SHOULD NOT contain properties which are not applicable to the corresponding _SkuPriceId_.
```

Comment:
Shouldn't this be a MUST NOT?
```suggestion
* The _SkuPriceDetails_ column MUST NOT contain properties which are not applicable to the corresponding _SkuPriceId_.
```

Reply:
I believe my thinking was that "properties which are not applicable" is loose enough of a statement that it would be hard to say (enforce) a MUST. If your VMs have a property about ephemeral disk, would that property be applicable to D8sv5 which does not have an ephemeral disk? I think that if the D8dsv5 had the property, I'd expect it to be on D8sv5 as well, even though the fact that it doesn't have one could be interpreted to mean that it's not applicable.

However, I sense you're reading this as something egregious like including a property communicating the number of cores on every resource (network, disks, PaaS, etc.). Planning to incorporate this change, but leaving the thread open for a little longer in case more people want to weigh in.